imod/mf6/utilities/dataset.py

```diff
@@ -46,3 +47,17 @@ def is_dataarray_none(datarray: Any) -> bool:
 def get_scalar_variables(ds: GridDataArray) -> list[str]:
     """Returns scalar variables in a dataset."""
     return [var for var, arr in ds.variables.items() if is_scalar(arr)]
+
+
+def assign_datetime_coords(
+    ds: GridDataArray, simulation_start_time: np.datetime64, time_unit: str = "d"
```

Comment:
'ds' is a dataarray. Therefor 'da' would be a better name

Reply:
done